docs/en-US/component/carousel.md

```diff
@@ -55,6 +55,18 @@ carousel/vertical
 
 :::
 
+## Card mode
```

Comment:
Maybe there is something wrong with my statement, I recommended to move the `Auto height demo` to line 38.

Reply:
ok,Is that so

<img width="1079" alt="image" src="https://user-images.githubusercontent.com/26755049/191651139-6fd2ca9a-a78e-4e74-bc30-26d11b06dafc.png">